### DISCUSSION TITLE
Contents of multi line template string gets parsed as markdown (v1)

### DISCUSSION THREAD

**Discussion by Stvad** [mdx-js/mdx#1485] [Q&A] [answered]
Hi I'm facing the issue described in #357. From conversation  in #195 it seems like it's fixed in v2 (?).  I'm using mdx though gatsby-plugin-mdx which still relies on v1.  

My question is - is there a way to work around this in v1? The only way I was able to come up so far is `string.replace(/\n+/g, '\n')` which kind of works but not ideal

**Comment by JounQin** [comment, marked as answer]
No, no new feature will be added into v1, and that's why we rewrite v2.

**Reply by Stvad** [reply]
I understand that, but what I'm asking is - if there is a workaround I can use? (escape those newlines somehow instead of deleting them for example)

**Reply by JounQin** [reply]
No.

**Reply by Stvad** [reply]
I see, that is unfortunate :(

**Reply by ChristianMurphy** [reply]
The solution would be to have Gatsby upgrade to MDX 2 or [XDM](https://github.com/wooorm/xdm)
